<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
	background-color: #ffdbcc;
}

QPushButton:pressed {
	background-color: #fcb9aa;
}
QPushButton {
	background-color: rgb(57, 208, 208);
	border-radius: 31px;
}</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите имя пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите фамилю</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_table">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
	background-color: #ffdbcc;
}

QPushButton:pressed {
	background-color: #fcb9aa;
}
QPushButton {
	background-color: rgb(57, 208, 208);
	border-radius: 31px;
}</string>
    </property>
    <property name="text">
     <string>Регистрация(right_table)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
